sdk/fido/fido_util.cpp

```diff
@@ -189,6 +189,10 @@ std::string to_zone_net_node(const wwiv::sdk::fido::FidoAddress& a) {
   return StrCat(a.zone(), ":", to_net_node(a));
 }
 
+std::string to_zone_net_node_point(const wwiv::sdk::fido::FidoAddress& a) {
```

Comment:
I'm not a guru at ftn either, but I *think* the msgid and origin line are only supposed to have a point if it's not zero.